src/screens/SignInV2/elements/SlideShowItem.js

```diff
@@ -90,7 +89,6 @@ export const SlideShowItem = ({
         handleLogin={handleLogin}
         isLogin={isLogin}
         text={text}
-        textSvg={textSvg}
         title={title}
         index={index}
         onNextSlide={__onNextSlide}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Here again, the `textSvg` prop has been removed. Make sure that this change doesn't break anything where this component is used.

```diff
- textSvg={textSvg}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->